main.go

```diff
@@ -117,9 +146,83 @@ func main() {
 		os.Exit(1)
 	}
 
+	go func() {
+		if err := runServer(context.Background(), db, setupLog, serverPort); err != nil {
+			setupLog.Error(err, "failed to run http server")
+			os.Exit(1)
+		}
+	}()
 	setupLog.Info("starting manager")
 	if err := mgr.Start(ctrl.SetupSignalHandler()); err != nil {
 		setupLog.Error(err, "problem running manager")
 		os.Exit(1)
 	}
 }
+
+// func getSecret(secretName string) []byte {
+// 	secret, err := ioutil.ReadFile(filepath.Join(dbSecretBasePath, secretName))
+// 	if err != nil {
+// 		setupLog.Error(err, "error reading secret")
+// 		os.Exit(1)
+// 	}
+// 	return secret
+// }
+
+// func getSecretFromFile(secretName string) (*corev1.Secret, error) {
+// 	data, err := ioutil.ReadFile(filepath.Join(secretBaseDir, secretName))
+// 	if err != nil {
+// 		return nil, err
+// 	}
+
+// 	return &corev1.Secret{
+// 		ObjectMeta: metav1.ObjectMeta{
+// 			Name: secretName,
+// 		},
+// 		Data: map[string][]byte{
+// 			secretName: data,
+// 		},
+// 	}, nil
+// }
```

Comment:
We can now get rid of this commented out code now that we're moving in a direction of accessing this information through environment variables.